packages/fxa-auth-server/lib/senders/emails/sass-compile-files.ts

```diff
@@ -0,0 +1,58 @@
+import sass from 'sass';
```

Comment:
In order to setup the directory structure that you laid out to me, I am using this script so as compile and generate stylesheets from multiple directories